colour/io/tests/test_tm2714.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from __future__ import annotations
+from copy import deepcopy
```

Comment:
We usually put a `from ... import` just after the regular `import`, here after `import os`